src/styles/mentors.scss

```diff
@@ -133,3 +133,13 @@
     transform: rotate(-45deg);
   }
 }
+
+.mentor-filter {
+  border-radius: 10px;
+  border: none;
+  width: auto;
+  height: 60px;
+  margin: 1em;
+  background-color: $teal-blue;
+  color: white;
```

Comment:
Oh - one more quick question: do you know why the text is black on one dropdown in the screenshot? I tried to recreate it myself to see what was up but they all look fine in my browser.

Reply:
No, unfortunately not :(

Reply:
I don't think I can actually reproduce it either